### ISSUE TITLE
LinkedIn Outreach

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#332]
Jesse Sookne initiated a LinkedIn outreach project on the a11y slack. Here is a link to that Slack thread: https://web-a11y.slack.com/archives/C0241L0TMGD/p1717526814051989

We'll participate in this outreach project. 

Here are some important notes:
- Link to Jesse's LinkedIn: https://www.linkedin.com/in/jessesookne/
- Link to Blake's LinkedIn: https://www.linkedin.com/in/blake1111/
- Link to Kevin's LinkedIn: https://www.linkedin.com/in/kevinjandrews/
- Jesse will post every weekday about web and mobile app accessibility, workplace accessibility, inclusive design, inclusive user research, explaining WCAG, explaining 
- Blake will post every Wednesday at 11:11AM about Automated Web Accessibility, Progress toward fixing 100M issues, progress making WordPress Accessible, Updates from Equalify, Inspiring Human Stories, Business Benefits of working on double bottom lines, Open Source
- We commit to working with our partner to promote each post within the first few minutes it goes up. Maybe we commit for a specific timeframe initially, e.g. for a month, so we can see how the process goes.
- We also jump in to promote posts from the other pairs as/when we can. No specific commitment here.

I'll continue to update this ticket as the project progresses.

**Comment by kevinandrews1** [comment]
Jesse, @bbertucc, and I met this afternoon. Very good discussion. Here's what happened:
- Engage immediately after a post is published to emphasize its importance.
- Encourage comments on new posts; LinkedIn's algorithm favors comments.
- Monitor dwell time (how long the post remains on screen) to gauge reading interest.
- Track profile views generated by posts; LinkedIn prefers posts that drive profile visits.
- Establish a consistent posting time.
- Use Buffer, if you want, to help automate post scheduling at specific times.
- Share insights and data about LinkedIn’s engagement strategies with the team as we proceed in the ocming weeks.
- Engagement responsibilities:
    - Kevin: Engage Monday and Wednesday, at least twice a week, at noon Eastern/11 Central.
    - Blake: Same.
    - Jesse: Engage five days a week.
- Blake will seek feedback on the a11y-marketing channel without requiring responses.

**Comment by bbertucc** [comment]
Note: updating my involvement from Tuesdays and Thursdays to just Wednesdays.

**Comment by bbertucc** [comment]
I'm going to close this. It's definitley a nice to have, but it's not pushing the needle.
